src/main/java/workout/workoutsession/WorkoutSession.java

```diff
@@ -52,10 +52,10 @@ public void workoutSessionStart() {
 
         while (!endWorkoutSession[0]) {
             try {
-                System.out.print("Workout Session >>>>> ");
+                WorkoutSessionUi.inputPrompt();
```

Comment:
Can create a new instance of workoutSessionUi and make the method non static. That would be more oop and better design